### PR TITLE
REL-3590: Add gender field and selector on the UI

### PR DESCRIPTION
**Pull request by cquiroz**
This PR adds a gender item to the p1 model and a UI to select it
Edit Principal Investigator 2019-07-28 18-51-18

### CODE REVIEW

File: bundle/edu.gemini.model.p1/src/test/resources/edu/gemini/model/p1/immutable/proposal_with_latin1_encoding.xml [hunk 1]
```diff
@@ -14,12 +14,13 @@
...
             <lastName>Investigator</lastName>
             <status>PhD</status>
+            <gender>None selected</gender>
             <email>kepler@if.ufrgs.br</email>
             <phone>+555133086556</phone>
             <address>
                 <institution>Universidade Federal do Rio Grande do Sul (UFRGS)</institution>
                 <address>Departamento de Astronomia - IF
-Av. Bento Gon�alves, 9500
+Av. Bento Gon�alves, 9500
```
Comment (sraaphorst) [inline bundle/edu.gemini.model.p1/src/test/resources/edu/gemini/model/p1/immutable/proposal_with_latin1_encoding.xml]: Gonçales looks like it got clobbered here.

**Comment by cquiroz** [comment]
I'll circulate the UI a bit before merging this

**Comment by sraaphorst** [comment]
Can we see it with the spacer? 😄

**Comment by cquiroz** [comment]
image

**Comment by sraaphorst** [comment]
Definitely looks better with the spacer. Thanks for satisfying my curiosity! 👍

**Pull request merged**